introduction/Organization.tex

```diff
@@ -6,39 +6,100 @@
 %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{Organization}
 
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Executive Committee
+%
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{Executive Committee --- ec@robocupathome.org}
 \label{sec:ec}
-The \iaterm{Executive Committee}{EC} consists of members of the board of trustees, and representatives of each activity area. Members representing the @Home league:
+The \iaterm{Executive Committee}{EC} consists of members of the board of trustees, and representatives of each activity area. The EC is responsible of the direction of the RoboCup@Home leagues, and has the last word in any decision taken. Is duty of the EC to instruct the new TC members when they join the committee, as well as instruct the OC Chair and supervise the decisions of both committees.
+
+\subsubsection{Members representing the @Home league}
 \begin{itemize}
   \item Kai Chen (University of Science and Technology of China, China)
   \item Dirk Holz (University of Bonn, Germany)
-  \item Caleb Rascon (Universidad Nacional Autonoma de Mexico, Mexico)
+  \item Caleb Rascon (Universidad Nacional Autónoma de México, Mexico)
   \item Sven Wachsmuth (Bielefeld University, Germany)
 \end{itemize}
 
+
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Technical Committee
+%
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{Technical Committee --- tc@robocupathome.org}
 \label{sec:tc}
-The \iaterm{Technical Committee}{TC} is responsible for the rules of each league. Members of the RoboCup@Home Technical Committee for \YEAR:
+The \iaterm{Technical Committee}{TC} is responsible for the rules of each league. Its responsibilities include:
+
+\begin{enumerate}
+  \item During preparation time
+  \begin{itemize}
+    \item Write and buff the rules in the rulebook
+    \item Design and write new tests (when applicable)
+    \item Evaluate previous performance and update tests accordingly
+  \end{itemize}
+  \item During the competition
+  \begin{itemize}
+    \item Direct Q\&A sessions during Team Leader meetings
+    \item Renounce to their teams and serves as referees (see \refsec{rule:referees})
```

Comment:
During competitions :-). 

TC members must not misuse their TC role to benefit their team (should be obvious though)